tests/main/snap-refresh-hold/task.yaml

```diff
@@ -11,6 +11,10 @@ restore: |
   snap refresh --unhold test-snapd-tools || true
   snap remove --purge test-snapd-tools || true
 
+debug: |
+  snap changes
+  snap changes | tail -n 4 | head -n 3 | awk '{ system("snap change " $1) }'
```

Comment:
Maybe this would benefit from a comment why we pick this exact change? Also I wonder if we could use the NR feature of awk to make it a bit shoter? Like
```
snap changes | awk '{if (NR == 3) system("snap change " $ 1) }
```
but that would require to know which line from the top to look at which we may not know?

Reply:
Not knowing awk, I'd do
```bash
snap changes | tail -n 4 | grep -P '^[0-9]+' | awk '{ system("snap change " $1) }'
```
but if that `NF != 0` trick works, I'm fine with that too. In any case I'd vote for a solution that doesn't require knowing the line numbers.

Reply:
No strong opinion, both suggestions look fine to me and maybe a small comment `// show the last 3 changes` or something?